docker/files/humble_base.repos

```diff
@@ -10,7 +10,7 @@ repositories:
   beluga:
     type: git
     url: https://github.com/Ekumen-OS/beluga.git
-    version: be0e09192217de4c53ecbadafaf5028405a249bc # glpuga/reduce_landmark_model_sensibility
+    version: main
```

Comment:
Hmm. On one hand, stability is good. On the other hand, I want us to be forced to keep this from breaking apart.

How about tying it to the [`v2` release branch](https://github.com/Ekumen-OS/beluga/tree/v2)?

Reply:
>  I want us to be forced to keep this from breaking apart.

If this break apart, we are unlikely to notice because it won't be us running it. Also:
- Once the image has been built, if main changes, the users won't notice because that layer will be cached and unchanged as far as docker is concerned.
- Bug reports, if any, will be smoother if we know what version the user's docker was built with. 

I'd rather have us periodically build this repo in CI every month or so with this line patched to "main" to make sure it still builds from scratch with the latest version, instead of using users as (frequently mute) canaries.

Anyway, I don't feel too strongly and it's not worth delaying this MR for this.

Reply:
Fair point. Pegging.

Reply:
Done in https://github.com/Ekumen-OS/beluga-demos/pull/8/commits/83fae496e0f3a96bdb42cfa7625a0e217d47897f